e2e/tests/global.setup.ts

```diff
@@ -19,33 +19,41 @@ setup('docker', async () => {
       const statusRegex = /(healthy|unhealthy|starting)/i; // "healthy"
 
       // Boolean filter to exclude empty strings
-      const containers = stdout.split('\n').filter(Boolean).map((container => {
-        const nameMatch = container.match(nameRegex);
-        const statusMatch = container.match(statusRegex);
-        
-        if (!nameMatch?.[0] || !statusMatch?.[1]) {
-          throw new Error('Failed to parse container info');
-        }
-        
-        const name = nameMatch[0];
-        const status = statusMatch[1];
-      
-        return {
-          name, status
-        }
-      }))
-      
-      
+      const containers = stdout
+        .split("\n")
+        .filter(Boolean)
+        .map((container) => {
+          const nameMatch = container.match(nameRegex);
+          const statusMatch = container.match(statusRegex);
+
+          if (!nameMatch?.[0] || !statusMatch?.[1]) {
+            // Exclude the act containers from the error
+            if (!container.includes("act-")) {
```

Comment:
I am confused where `act-` containers are coming from?
Are those on your local machine?

If those are getting into the mix I think you can just update the `nameRegex` to be something like
`^famf-.*$`

Reply:
Btw for this I found the `docker compose` has a `--wait` and `--wait-timeout (seconds)` option that could get rid of this setup test file.

Those options will wait for a healthy/running status of the container before detaching when using `make dev-background`

Would look like
```make
dev-background: build-dev
	docker compose -p famf -f ./docker/${COMPOSE_FILE} up -d --wait --wait-timeout 120
```